code/infrastructure-api/manifests/rstudio.deployment.template.yml

```diff
@@ -27,32 +65,23 @@ spec:
             secretKeyRef:
               name: {{ name }}
               key: password
-        volumeMounts:
-        - name: glusterfsvol
-          mountPath: /data
         resources:
           requests:
             cpu: 200m
             memory: 128Mi
           limits:
             cpu: 2
             memory: 4Gi
-      - name: {{ name }}-connect
-        image: {{ &rstudio.connectImageName }}:{{ rstudio.connectVersion }}
-        imagePullPolicy: IfNotPresent
-        ports:
-        - containerPort: 8000
-          protocol: TCP
-        env:
-        - name: CONNECT_TYPE
-          value: RSTUDIO
-        livenessProbe:
-          httpGet:
-            path: /status
-            port: 8000
-          initialDelaySeconds: 5
-          periodSeconds: 10
+{{#volumeMount}}
+        volumeMounts:
+        - name: glusterfsvol
+          mountPath: /data
+        - name: homedir
+          mountPath: /home
       volumes:
       - name: glusterfsvol
         persistentVolumeClaim:
-          claimName: {{ datalabVolume }}-claim
+          claimName: {{ volumeMount }}-claim
+      - name: homedir
```

Comment:
Doesn't this make the homedir disposable? Is that what we want? How is that different to not mounting anything?